syntax = "proto3";

message StockRequest { string StockName = 1; }

message Stock {
    string StockName = 1;
    string StockName = 1;
    float price = 2;
}

service StockBot {
    rpc GetStock(StockRequest) returns (Stock) {}
}
